lib/rules/sort-object-props.js

```diff
@@ -18,24 +20,13 @@ module.exports = {
         return {
             ObjectExpression: function(node) {
                 node.properties.reduce(function(prev, current) {
-                    if (opts.ignoreMethods && current.value.type === "FunctionExpression") {
-                        return current;
```

Comment:
Please review this (ignoreMethods flag and value.type), though I've implemented it, I wasn't using this as a feature before, so please double check. A test for this currently doesn't exist